### DISCUSSION TITLE
Favourites: cannot see them

### DISCUSSION THREAD

**Discussion by adm2k** [FreshRSS/FreshRSS#5365] [Q&A]
Favorites show two on left panel but none on the right panel.

![image](https://user-images.githubusercontent.com/120423301/235373354-9132e4f8-814b-4310-8b6c-af4db5405279.png)

**Comment by math-GH** [comment]
Hi, it is the same like in you previous question #5356

You have 2 articles marked as favourites. (see the number in the left hand side panel)

0 favourites articles are unread.

If you want to see also the already read articles, then you need to activate the "show read articles" button.

**Comment by math-GH** [comment]
Maybe this config could be better for you:

(1) regarding #5356
(2) would fix this question here

![grafik](https://user-images.githubusercontent.com/1645099/235377907-14db67a2-4a4f-4102-90f2-10c971929d99.png)
